### ISSUE TITLE
Connecting to service bus with managed identity auth

### ISSUE THREAD

**Issue by ancientpanda** [paolosalvatori/ServiceBusExplorer#714]
I can only connect to the service bus explorer using shared conn strings, is there a way with managed auth?

**Comment by janv8000** [comment]
Related to https://github.com/paolosalvatori/ServiceBusExplorer/issues/607 ?

**Comment by ErikMogensen** [comment]
> is there a way with managed auth?

No, but it is something that many have requested so it is one our main priorities.

> Related to #607 ?

Yes. Definitely.

**Comment by Frulfump** [comment]
https://github.com/paolosalvatori/ServiceBusExplorer/issues/607 is now closed as completed!
